lib/galaxy/webapps/galaxy/controllers/visualization.py

```diff
@@ -404,7 +404,7 @@ def imp(self, trans, id):
             # Redirect to load galaxy frames.
             return trans.show_ok_message(
                 message="""Visualization "%s" has been imported. <br>You can <a href="%s">start using this visualization</a> or %s."""
-                % (visualization.title, web.url_for(controller='visualization'), referer_message), use_panels=True)
+                % (visualization.title, web.url_for(controller='visualizations/list'), referer_message), use_panels=True)
```

Comment:
Since it's not actually a controller endpoint anymore, this should probably just be `web.url_for('visualizations/list')`, right?

Reply:
Currently, we usually refer to the client routes also as controllers. Changing this without resetting the controller would redirect to `visualization/visualizations/list` right?

Reply:
`/visualizations/list` and not `visualizations/list`, I should have said, which hopefully has the right behavior?

Reply:
I changed it for this particular case. I mean that we usually kept the form `controller=ROUTE` in the backend even when we transitioned them to front end routes.

Reply:
@guerler Got it, thanks.  We should dig into this more, later, and see what the pros and cons are.  To me, it doesn't seem like there's a need to use the controller/action form when there aren't actual controllers to represent; just specific routes -- my concern (maybe unfounded, though, we should look) was that Routes is going to make extra assumptions based on the controller argument vs absolute paths.